packages/cyberstorm/src/components/Layout/Teams/TeamSettings/TeamDetails/TeamDetails.tsx

```diff
@@ -76,12 +76,12 @@ export function TeamDetails(props: Props) {
         <SettingItem
           title="Profile Summary"
           description="A short description shown in header and profile cards"
-          content={<TextInput placeHolder={teamData.description} />}
+          content={<TextInput placeholder={teamData.description} />}
```

Comment:
Is this actually a placeholder? Looks more like the proper value, prefilled into the form? Comment applies to other changes in this file as well.